pkg/admin/constant/kubenetes.go

```diff
@@ -0,0 +1,25 @@
+package constant
+
+const baseLabel = "dubbo.io"
+
+const (
+	ApplicationLabel = baseLabel + "/application"
+	ServiceKeyLabel  = baseLabel + "/serviceKey"
+	GroupLabel       = baseLabel + "/group"
+	VersionLabel     = baseLabel + "/version"
```

Comment:
Revisit `label name` later.

Reply:
The label name and base prefix should be synchronized with the "best practices" in the documentation.